docs/api.rst

```diff
@@ -2812,9 +2814,9 @@ of :class:`enum.Enum`.
         - :attr:`~AuditLogDiff.description`
         - :attr:`~AuditLogDiff.channel`
         - :attr:`~AuditLogDiff.privacy_level`
-        - :attr:`~AuditLogDiff.location`
-        - :attr:`~AuditLogDiff.status`
-        - :attr:`~AuditLogDiff.location_type`
+        - :attr:`~discord.ScheduledEvent.location`
+        - :attr:`~discord.ScheduledEvent.status`
+        - :attr:`~discord.ScheduledEventLocation.type`
```

Comment:
Why was this changed?

Reply:
These were broken references due to AuditLogDiff not having those attributes.

Reply:
Oh ok, I guess they could be documented in another PR